src/components/Navbar/Navbar.style.js

```diff
@@ -7,7 +7,7 @@ export const NavbarContainer = styled.nav`
   width: 95%;
   height: 80px;
   background-color: ${darkModeColors.background};
-  height: ${(props) => (props.extendNavbar ? '100vh' : '80px')};
+  height: ${(props) => (props.extendNavbar ? '400px' : '80px')};
```

Comment:
I am not familiar with css.
do you consider `px`, `rem`, `vh` and `%` in your css?
why use px in this case?

Reply:
Hi, that is a good thing to take into consideration. When to use px or rem values